spec.emu

```diff
@@ -275,12 +276,97 @@
       <emu-note>That means that "A" (`LATIN CAPITAL LETTER A`) measures as 1 code unit, and "🔥" (`FIRE`) measures as 2 code units. Source maps for other content types may diverge from this.</emu-note>
     </dd>
   </dl>
+
+  <emu-clause id="sec-vlq-grammar">
+    <h1>base64 VLQ grammar</h1>
+    <p>A base64-encoded variable-length quantity adheres to the following lexical grammar:</p>
+
+    <emu-grammar type="definition">
+      Vlq ::
+        VlqDigitList
+
+      VlqDigitList ::
+        TerminalDigit
+        ContinuationDigit VlqDigitList
+
+      TerminalDigit ::
+        OneOfTerminalDigit
+
+      ContinuationDigit ::
+        OneOfContinuationDigit
+
+      OneOfTerminalDigit :: one of
+        `A` `B` `C` `D` `E` `F` `G` `H` `I` `J` `K` `L` `M` `N` `O` `P` `Q` `R`
+        `S` `T` `U` `V` `W` `X` `Y` `Z` `a` `b` `c` `d` `e` `f`
+
+      OneOfContinuationDigit :: one of
+        `g` `h` `i` `j` `k` `l` `m` `n` `o` `p` `q` `r` `s` `t` `u` `v` `w` `x`
+        `y` `z` `0` `1` `2` `3` `4` `5` `6` `7` `8` `9` `+` `/`
+    </emu-grammar>
+
+    <emu-note>
+      |OneOfTerminalDigit| and |OneOfContinuationDigit| will go away and are only present in the preview draft. They are here because ecmarkup currently does not support *one of* productions in SDOs.
+    </emu-note>
+
+    <emu-clause id="sec-VLQSignedValue" type="sdo">
+      <h1>VLQSignedValue ( ): an integer</h1>
+      <dl class="header"></dl>
+      <emu-grammar>
+        Vlq :: VlqDigitList
+      </emu-grammar>
+      <emu-alg>
+        1. Let _unsigned_ be the VLQUnsignedValue of |VlqDigitList|.
+        1. If _unsigned_ modulo 2 = 1, let _sign_ be -1.
+        1. Else, let _sign_ be 1.
+        1. Return _sign_ × floor(_unsigned_ / 2).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-VLQUnsignedValue" type="sdo">
+      <h1>VLQUnsignedValue ( ): an non-negative integer</h1>
+      <dl class="header"></dl>
+      <emu-grammar>
+        VlqDigitList ::
+          TerminalDigit
+      </emu-grammar>
+      <emu-alg>
+        1. Return VLQUnsignedValue of |TerminalDigit|.
+      </emu-alg>
```

Comment:
This is actually not needed, as the default behavior for SDOs is to apply itself on the only child: https://tc39.es/ecma262/#sec-algorithm-conventions-syntax-directed-operations (from "Unless explicitly specified otherwise...").

Reply:
Done.